src/app/hooks/useClickTrackerHandler/index.jsx

```diff
@@ -9,29 +10,55 @@ import { ServiceContext } from '../../contexts/ServiceContext';
 import { isValidClick } from './clickTypes';
 
 const EVENT_TYPE = 'click';
+export const LITE_ATI_TRACKING = 'data-lite-ati-tracking';
+
+const useExtractTrackingProps = (props = {}) => {
+  const eventTrackingContext = useContext(EventTrackingContext);
+
+  const { componentName, url, advertiserID, format, detailedPlacement } = props;
+  const { pageIdentifier, platform, producerId, statsDestination } =
+    eventTrackingContext;
+
+  const campaignID = props?.campaignID || eventTrackingContext?.campaignID;
+
+  return {
+    pageIdentifier,
+    producerId,
+    platform,
+    statsDestination,
+    componentName,
+    campaignID,
+    format,
+    type: EVENT_TYPE,
+    advertiserID,
+    url,
+    detailedPlacement,
+  };
+};
 
 const useClickTrackerHandler = (props = {}) => {
+  const {
+    pageIdentifier,
+    producerId,
+    platform,
+    statsDestination,
+    componentName,
+    campaignID,
+    format,
+    advertiserID,
+    url,
+    detailedPlacement,
+  } = useExtractTrackingProps(props);
+
   const preventNavigation = props?.preventNavigation;
-  const componentName = props?.componentName;
-  const url = props?.url;
-  const advertiserID = props?.advertiserID;
-  const format = props?.format;
   const optimizely = props?.optimizely;
   const optimizelyMetricNameOverride = props?.optimizelyMetricNameOverride;
-  const detailedPlacement = props?.detailedPlacement;
 
   const { trackingIsEnabled } = useTrackingToggle(componentName);
   const [clicked, setClicked] = useState(false);
   const eventTrackingContext = useContext(EventTrackingContext);
 
-  const {
-    pageIdentifier,
-    platform,
-    producerId,
-    producerName,
-    statsDestination,
-  } = eventTrackingContext;
-  const campaignID = props?.campaignID || eventTrackingContext?.campaignID;
+  const { producerName } = eventTrackingContext;
```

Comment:
Should we have included this in the extractTrackingProps function, given that it has a handle on the eventTrackingContext? (I know it's not used for the lite click tracking, but it is used for non-lite click tracking.